contrib/verify-commits/trusted-keys

```diff
@@ -1,8 +1,5 @@
 71A3B16735405025D447E8F274810B012346C9A6
 1F4410F6A89268CE3197A84C57896D2FF8F0B657
```

Comment:
`gpg --list-key --fingerprint 1F4410F6A89268CE3197A84C57896D2FF8F0B657` returns revoked key by @sipa. Once the commits signed by this key are buried under the trusted root, you could get rid of those lines in allow-revsig-commits.